docs/web_reference.rst

```diff
@@ -1128,6 +1135,13 @@ duplicated like one using :meth:`Application.copy`.
       :param kwargs: additional parameters for :class:`RequestHandlerFactory`
                      constructor.
 
+      .. note::
```

Comment:
The best solution is explicitly document all available parameters.
It makes sense not only for getting rid of weird deprecation.
I suspect the most part of users just don't know what parameters are available.

But it's a subject for another PR.
Feel free to create it if you want.